tutorials/katacoda/thanos/7-multi-tenancy/step2.md

```diff
@@ -46,14 +46,16 @@ So why not we start something like this in front of our "Tomato" Querier?
 ```
 docker run -d --net=host --rm \
     --name prom-label-proxy \
-    quay.io/thanos/prom-label-proxy:v0.3.0-rc.0-ext1 \
+    quay.io/prometheuscommunity/prom-label-proxy:v0.3.0 \
     -label tenant \
     -upstream http://127.0.0.1:29090 \
     -insecure-listen-address 0.0.0.0:39090 \
-    -non-api-path-passthrough \
-    -enable-label-apis && echo "Started prom-label-proxy"
+    -enable-label-apis  \
+-unsafe-passthrough-paths=/api/v1/stores,/api/v1/status/flags,/graph,/status,/api/v1/status/buildinfo,/api/v1/status/runtimeinfo,/flags,/static/css/2.a92efa4c.chunk.css,/static/css/main.e1aaea6d.chunk.css,/static/js/2.2de83b5e.chunk.js,/static/js/main.a767bdbf.chunk.js,/manifest.json && echo "Started prom-label-proxy"
```

Comment:
This is nasty - see https://github.com/prometheus-community/prom-label-proxy/issues/68